Zend/zend_inheritance.c

```diff
@@ -1158,7 +1158,7 @@ ZEND_API void zend_do_inheritance_ex(zend_class_entry *ce, zend_class_entry *par
 			do {
 				dst--;
 				src--;
-				ZVAL_COPY_VALUE(dst, src);
+				*dst = *src; /* Copy Z_EXTRA as well */
```

Comment:
Here and later: better to keep the existing code under #if 0 with comments about optimization (or introduce special macro ZVAL_COPY_PROP_VALUE)